stdlib/public/SDK/Foundation/Foundation.swift

```diff
@@ -984,6 +996,24 @@ extension NSDictionary : Sequence {
   }
 }
 
+extension NSMutableDictionary {
+  // Bridging subscript.
+  @nonobjc
+  final public subscript(key: _Hashable) -> Any? {
```

Comment:
Please add a `FIXME(ABI)` with a note that it should become a generic subscript.